cypress/e2e/tests/administration/jira-connection/sort.test.ts

```diff
@@ -0,0 +1,92 @@
+import {
+    clickOnSortButton,
+    createMultipleJiraConnections,
+    deleteByList,
+    getTableColumnData,
+    login,
+    verifySortAsc,
+    verifySortDesc,
+} from "../../../../utils/utils";
+import { Jira } from "../../../models/administration/jira-connection/jira";
+import { JiraCredentials } from "../../../models/administration/credentials/JiraCredentials";
+import { getJiraCredentialData } from "../../../../utils/data_utils";
+import { CredentialType, instanceName, SEC, SortType } from "../../../types/constants";
+import { jiraTable } from "../../../views/jira.view";
+
+describe(["@tier2"], "Jira connections sort validations", function () {
+    const useTestingAccount = true;
+    let jiraBasicCredential: JiraCredentials;
+    let jiraConnectionList: Jira[];
+    let isInsecure = false;
+    let toBeCanceled = true;
+    let expectedToFail = true;
```

Comment:
we are starting to not put validations in models anymore . we can refactor that in another PR.

Reply:
Sure